ThirdPartyNotices.txt

```diff
@@ -122,3 +123,31 @@ OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 THE SOFTWARE.
 ============================================================
 END OF DefinitelyTyped NOTICES AND INFORMATION
+
+%% roslyn-sdk NOTICES AND INFORMATION BEGINS HERE
+============================================================
```

Comment:
This is for the syntax tree icon, which is from https://github.com/dotnet/roslyn-sdk/blob/master/src/VisualStudio.Roslyn.SDK/SyntaxVisualizer/Roslyn.SyntaxVisualizer.Extension/SyntaxTree.ico